File: src/circleannotation.go
// circleannotation.go
//
// Copyright (c) 2026 PDFjet Software
// Licensed under the MIT License. See LICENSE file in the project root.

package pdfjet

// CircleAnnotation extends BaseAnnotation to represent a circle.
type CircleAnnotation struct {
	BaseAnnotation
}

// NewCircleAnnotation acts as the constructor.
func NewCircleAnnotation() *CircleAnnotation {
	c := &CircleAnnotation{BaseAnnotation: *newBaseAnnotation()}
	c.BaseAnnotation.annotationType = annotationCircle
	return c
}

File: Sources/PDFjet/CircleAnnotation.swift
/**
 * CircleAnnotation.swift
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
import Foundation

/// A circle annotation.
public class CircleAnnotation: BaseAnnotation {
    override public init() {
        super.init()
        self.annotationType = Annotation.Circle
    }
}
